### PR TITLE
Feature/AA 409 Pending Supplier Reference and Confirmed Bookings Reports

### PR DESCRIPTION
**Pull request by konstyefremov**
https://github.com/happy-travel/agent-app-project/issues/409

Added two reports. Need to check if I understood the applied filters correctly.

Also, these reports mostly copy each other with the only difference in the 'where' block. What would be a better way to keep it DRY?

### CODE REVIEW

**Comment by mingazhev** [comment]
Please fix the conflicts

**Review by mingazhev — COMMENTED** [review]
Need conflicts fix

**Review by kirillta — APPROVED** [review]

File: Api/Infrastructure/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -778,6 +778,16 @@ public static IServiceCollection AddServices(this IServiceCollection services)
             services.AddTransient<IRecordManager<SupplierWiseRecordData>, SupplierWiseRecordsManager>();
             services.AddTransient<IRecordManager<FullBookingsReportData>, FullBookingsRecordManager>();
             services.AddTransient<IRecordManager<SalesBookingsReportData>, SalesBookingsRecordManager>();
+            services.AddTransient<IConverter<AgencyWiseRecordData, AgencyWiseReportRow>, AgencyWiseRecordDataConverter>();
```
Comment (mingazhev) [inline Api/Infrastructure/ServiceCollectionExtensions.cs:781]: Consider moving all report needed registrations to a separate method

**Review by mingazhev — APPROVED** [review]
With a small comment

**Pull request merged**